src/background.js

```diff
@@ -300,6 +347,10 @@ async function enroll() {
     queryScorer.addDocument({ id: docID, words: KEYWORDS[docID] });
   }
 
+  // Trigger a browser update check.  (This won't actually check if updates are
+  // disabled for some reason, e.g., by policy.)
```

Comment:
I don't recall the story behind this, we should always respect policies. Hopefully it's not a big deal considered one of the policies will very likely disable experiments and these are not running on ESR.